README.md

```diff
@@ -9,23 +7,21 @@
 
 ## Multi-cluster Configuration Management for Edge, Multi-Cloud, and Hybrid Cloud
 
-[![](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://www.firsttimersonly.com/)&nbsp;&nbsp;&nbsp;
+[![](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://www.firsttimersonly.com/)   
```

Comment:
Please leave the source in ASCII, do not replace `&nbsp;` with a non-breaking space character (U+00A0).